[Feature] Remove update-maxpods from Makefile

<!--
Thank you for opening this feature request! You are helping make eksctl a better project :)
In order to help us process this request faster, please provide as much detail as possible.

Before creating a feature request, please search existing issues to see if this request has already been made. If there is a similar feature request please up-vote it and/or add your comments to it instead.

For more information on how to contribute to eksctl, please refer to our contributing docs.
https://github.com/weaveworks/eksctl/blob/main/CONTRIBUTING.md#opening-issues
-->

### What feature/behavior/change do you want?
<!--
A clear and concise description of the change you would like to see in eksctl.

Do not hesitate, when appropriate, to share the exact commands or API you would like,
and/or to share a diagram (e.g.: asciiflow.com): "a picture is worth a thousand words".
-->
Hello, I have noticed that the file `maxpods_generate.go` has been removed, so I was wondering if this `make` command should be removed or changed in some way?
https://github.com/weaveworks/eksctl/blob/main/Makefile#L163
```
### Update maxpods.go from AWS
.PHONY: update-maxpods
update-maxpods: ## Re-download the max pods info from AWS and regenerate the maxpods.go file
	@cd pkg/nodebootstrap && go run maxpods_generate.go
```

### Why do you want this feature?
<!--
A clear and concise description of why you want this feature.
What is it you are trying to accomplish?
Include use-cases and any specific constraints you may have.
-->
I noticed when running the command locally, I receive the following error message, which is why I was not sure if this `make` command is still valid?
```
% make update-maxpods
stat maxpods_generate.go: no such file or directory
make: *** [update-maxpods] Error 1
```

Any information, or help, would be greatly appreciated. 😄

--- a/Makefile
+++ b/Makefile
@@ -157,11 +157,6 @@ generate-all: generate-always $(conditionally_generated_files) ## Re-generate al
 check-all-generated-files-up-to-date: generate-all ## Run the generate all command and verify there is no new diff
 	git diff --quiet -- $(conditionally_generated_files) || (git --no-pager diff $(conditionally_generated_files); echo "HINT: to fix this, run 'git commit $(conditionally_generated_files) --message \"Update generated files\"'"; exit 1)
 
-### Update maxpods.go from AWS
-.PHONY: update-maxpods
-update-maxpods: ## Re-download the max pods info from AWS and regenerate the maxpods.go file
-	@cd pkg/nodebootstrap && go run maxpods_generate.go
-
 ### Update aws-node addon manifests from AWS
 pkg/addons/default/assets/aws-node.yaml:
 	go generate ./pkg/addons/default/aws_node_generate.go

